keps/sig-node/606-compute-device-assignment/README.md

```diff
@@ -148,6 +173,20 @@ We have identified two main ways of testing this API:
 E2E tests are explicitly not written because they would require us to generate and deploy a custom container.
 The infrastructure required is expensive and it is not clear what additional testing (and hence risk reduction) this would provide compare to node e2e tests.
 
+##### Prerequisite testing updates
+
+##### Unit tests
+
+- `k8s.io/kubernetes/pkg/kubelet/apis/podresources`: `20230127` - `61.5%`
+
+##### Integration tests
+
+covered by e2e tests
+
+##### e2e tests
+
+- `k8s.io/kubernetes/test/e2e_node/podresources_test.go`: https://storage.googleapis.com/k8s-triage/index.html?test=POD%20Resources
+
 ### Graduation Criteria
```

Comment:
sorry for the late feedback. I see an open PR to add Windows support. Should we add Windows to be a graduation criteria? I don't see any blockers there and should be straightforward thing to add

Reply:
I am ok to update this after the merging.

Reply:
AFAICT support for all the platforms was a implicit requirement, but in hindisight is indeed better to mention it explicitely, thanks for pointing this out.